Marlin/src/HAL/HAL_AVR/HAL.h

```diff
@@ -365,6 +365,7 @@ inline void HAL_adc_init() {
   #define HAL_START_ADC(pin) ADCSRB = 0; SET_ADMUX_ADCSRA(pin)
 #endif
 
+#define HAL_ADC_RESOLUTION 1024 // 10-bit
```

Comment:
Also I think this can be a effective bit count ("10") instead of bit mask/max value/adc counts ("1024") because 1024 can be easy get from 10 by bit-shifting.

Reply:
Unsigned int multiplication by powers of 2 should decompose to a bit-shift, but we can look at the compiler output to be certain.